app/graphql/albums/resolver.js

```diff
@@ -0,0 +1,11 @@
+const { getAlbum, getAlbumPhotos } = require('../../services/album');
```

Comment:
I would do this directly in the resolver, in `albums/queries`. Logic is very simple in order to move it to another file right now.

Reply:
it is understood